src/App.css

```diff
@@ -1,38 +1,266 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;1,100;1,200;1,300;1,400&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat;
+  background-color: #f5f6fa;
+}
+
+header {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  padding: 1rem 5rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
```

Comment:
- [ ] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

no repeating of the same rule
Please correct those issues and send another review request, good luck, and well done so far!